<launch>
  <arg name="robot_namespace" default="robot0" />
  <arg name="robot_radius" default="0.2"/>
  <arg name="num_obsts" default="2"/>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find nav_configs)/config/custom_teb_rosconsole.conf"/>

  <node pkg="modify_stdr_scan" type="modify_stdr_scan" name="modify_stdr_scan" output="log" args="$(arg robot_namespace) $(arg robot_radius)" />

  <node name="egocircle_node" pkg="egocircle" type="ego_circle">
      <param name="base_frame_id" value="$(arg robot_namespace)"/>
      <param name="odom_frame_id" value="map_static"/>
      <remap from="/scan" to="$(arg robot_namespace)/mod_laser_0"/>
  </node>

  <node pkg="gap_estimation" type="gap_estimation" name="gap_estimation" output="log" args="$(arg robot_namespace) $(arg robot_radius)" />

  <arg name="model" default="STDR" />
  <param name="model" value="$(arg model)"/>

  <node pkg="turtlebot3_teleop" type="turtlebot3_teleop_key" name="turtlebot3_teleop_keyboard"  output="screen">

	<remap from="/cmd_vel" to="/robot2/cmd_vel"/>
  </node>

   <node type="rviz" name="rviz" pkg="rviz" args="-d $(find dynamic_gap)/config/dgap.rviz"/>
</launch>
